.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: ruby
+rvm:
+  - 2.5.1
+
+env:
+  global:
+    - RUBY_GC_MALLOC_LIMIT=90000000
+    - RUBY_GC_HEAP_FREE_SLOTS=200000
+
+services:
+  - postgresql
+
+before_install:
```

Comment:
You can keep this line because in few card we will add some specs.
- "echo '--colour' > ~/.rspec"